app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from os import remove, path, makedirs
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_input, path_info, file_name = command.split()
```

Comment:
To not perform `split` operation twice, you could write it's result to a separate variable in advance
```suggestion
    command_slpit = command.split()
    if len(command_slpit) == 3:
        command_input, path_info, file_name = command_slpit
```